### DISCUSSION TITLE
增加“回到旧版”功能

### DISCUSSION THREAD

**Discussion by WinterSamu** [the1812/Bilibili-Evolved#4045] [功能建议 / Ideas]
这里的回到旧版不是回到14，15年，而是以下的样子：（第一张图片）
![SharedScreenshot1](https://user-images.githubusercontent.com/103006798/222942225-d7a95838-3a5c-4869-bdec-abe7b2115ad3.jpg)

为什么要建议这个功能，因为在播放收藏夹页面和番剧播放页面会是以下的样式（说实话：这新UI页面设计的太烂了，都要成优爱腾了）：（第二张图片）
![SharedScreenshot](https://user-images.githubusercontent.com/103006798/222942160-e5722cac-1780-46d6-b753-b0c7339885c0.jpg)

而且也有许多用户反应他们的“回到旧版”已经不见了。
希望尽快添加这个功能。谢谢！^_^

**Comment by the1812** [comment]
b 站迟早会砍掉，我可不是网页时光机

**Comment by WinterSamu** [comment]
啊（失落），抱歉啊，但还是谢谢啊。

那么可以暂时出个脚本维持一下cookie吗？现在还可以通过cookie维持一下。

总之还是谢谢了。

从 Windows 版邮件发送

发件人: Grant Howard
发送时间: 2023年3月5日 12:58
收件人: the1812/Bilibili-Evolved
抄送: WinterSamu; Author
主题: Re: [the1812/Bilibili-Evolved] 增加“回到旧版”功能 (Discussion #4045)

b 站迟早会砍掉，我可不是网页时光机

**Reply by Tinhone** [reply]
https://greasyfork.org/zh-CN/scripts/455571
(打个广告，或许能帮到您
( ﾟ∀。)

**Comment by WinterSamu** [comment]
脚本不错，但是番剧页和收藏夹连续播放页确实恢复不了了T_T

逼站设计的什么破UI，还强迫给用户喂shi。（怒）

从 Windows 版邮件发送

发件人: Tinhone
发送时间: 2023年3月5日 15:45
收件人: the1812/Bilibili-Evolved
抄送: WinterSamu; Author
主题: Re: [the1812/Bilibili-Evolved] 增加“回到旧版”功能 (Discussion #4045)

https://greasyfork.org/zh-CN/scripts/455571
(打个广告，或许能帮到您
( ﾟ∀。)
